net/net-mysql-client.cpp

```diff
@@ -271,10 +271,12 @@ static int sqlc_inner_authorise (struct connection *c) {
 
   T = (struct mysql_auth_packet_end *)p;
 
-  const char *sql_username = SQLC_FUNC(c)->sql_get_username(c);
   const char *sql_database = SQLC_FUNC(c)->sql_get_database(c);
+  const char *sql_user = SQLC_FUNC(c)->sql_get_user(c);
   const char *sql_password = SQLC_FUNC(c)->sql_get_password(c);
 
+  fprintf(stdout, "mysql db: %s; user: %s; password: *\n", sql_database, sql_user);
```

Comment:
Is this intended to stay here or it's a debug leftover?

Reply:
I think it's useful information when use custom db settings, but of course, it's not required